components/menu/index.css

```diff
@@ -10,170 +10,595 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+  grid-area: valueArea;
+}
+
+.spectrum-Menu-itemActions {
+  grid-area: actionsArea;
+}
+
+.spectrum-Menu-chevron {
+  grid-area: chevronArea;
+  align-self: center;
+  block-size: var(--spectrum-menu-item-checkmark-height);
+  height: var(--spectrum-menu-item-checkmark-height);
+  inline-size: var(--spectrum-menu-item-checkmark-width);
+  width: var(--spectrum-menu-item-checkmark-width);
+}
+
+.spectrum-Menu-item--collapsible .spectrum-Menu-chevron {
+  grid-area: chevronAreaCollapsible;
+}
+
+.spectrum-Menu-sectionHeading {
+  grid-area: sectionHeadingArea;
+  grid-column: 1 / -1;
+}
+
+.spectrum-Menu-item--collapsible .spectrum-Menu-sectionHeading {
+  /*
+    start at 5th column, to account for icons & match sectionHeadingArea
+    ". chevronAreaCollapsible . iconArea sectionHeadingArea  .  .  ."
+    stretch to last column
+  */
+  grid-column: 5 / -1;
+}
+
+.spectrum-Menu li:not(.spectrum-Menu-item, .spectrum-Menu-divider) > .spectrum-Menu-sectionHeading {
+  padding-inline-start: var(--mod-menu-item-label-inline-edge-to-content, var(--spectrum-menu-item-label-inline-edge-to-content));
+}
+
+.spectrum-Menu-itemDescription {
+  grid-area: descriptionArea;
+}
+
+.spectrum-Menu-item--collapsible .spectrum-Menu,
+.spectrum-Menu-item .spectrum-Menu {
+  grid-area: submenuArea;
+  grid-column: 1 / -1;
+}
+
+.spectrum-Menu-item--drillIn .spectrum-Menu-chevron {
+  grid-area: chevronAreaDrillIn;
+}
+
+.spectrum-Menu-itemIcon:not(.spectrum-Menu-chevron, .spectrum-Menu-checkmark) {
+  block-size: var(--mod-menu-item-icon-height, var(--spectrum-menu-item-icon-height));
+  inline-size: var(--mod-menu-item-icon-width, var(--spectrum-menu-item-icon-width));
+}
+
+.spectrum-Menu-checkmark {
+  block-size: var(--mod-menu-item-checkmark-height, var(--spectrum-menu-item-checkmark-height));
+  inline-size: var(--mod-menu-item-checkmark-width, var(--spectrum-menu-item-checkmark-width));
+  /* default: checkmarks that have text directly next to them */
+  margin-inline-end: var(--mod-menu-item-text-to-control, var(--spectrum-menu-item-text-to-control));
+}
+
+/* checkmarks that have an icon directly next to them */
+.spectrum-Menu-checkmark--withAdjacentIcon {
+  margin-inline-end: var(--mod-menu-item-text-to-control, var(--spectrum-menu-item-text-to-control));
+}
+
+/* always provide space at the end of a workflow icon */
+.spectrum-Menu-itemIcon--workflowIcon {
+  margin-inline-end: var(--mod-menu-item-label-text-to-visual, var(--spectrum-menu-item-label-text-to-visual));
+}
+
+.spectrum-Menu-chevron {
+  /* default: chevrons (aka collapsibles) that have no icon and instead have text directly next to them */
+  margin-inline-end: var(--mod-menu-item-text-to-control, var(--spectrum-menu-item-text-to-control));
+}
+
+/* chevrons (aka collapsibles) that have an icon directly next to them */
+.spectrum-Menu-chevron--withAdjacentIcon {
+  margin-inline-end: var(--mod-menu-item-text-to-control, var(--spectrum-menu-item-text-to-control));
```

Comment:
If this value is the same as the value for the above `.spectrum-Menu-chevron`, couldn't we just get rid of this `--withAdjacentIcon` entirely? Same goes for `spectrum-Menu-checkmark--withAdjacentIcon`

Reply:
👍 I've removed these classes after double-checking with Patrick. They don't appear to be needed anymore.